CHANGELOG.md

```diff
@@ -2,6 +2,106 @@
 
 ## Unreleased
 
+This release is a notable release for metrics:
+
+* The base2 exponential histogram aggregation has been marked as stable. To use, configure
+  your `MetricExporter` with a default aggregation
+  of `Aggregation.base2ExponentialBucketHistogram()` for histogram instruments. If using OTLP
+  exporter with autoconfigure,
+  set `OTEL_EXPORTER_OTLP_METRICS_DEFAULT_HISTOGRAM_AGGREGATION=BASE2_EXPONENTIAL_BUCKET_HISTOGRAM`.
+  If using configuring OTLP programatically,
+  use `Otlp*MetricExporterBuilder#setDefaultAggregationSelector(DefaultAggregationSelector)`.
+* The metrics SDK undergone significant internal refactoring which results in reduced complexity and
+  significantly reduced memory allocation during metric collection / export.
+
+### API
+
+#### Logs
+
+* WARNING: Split out Event API from Log API. Event API is now published in `opentelemetry-api-logs`.
```

Comment:
Shouldn't that be `opentelemetry-api-events`?
```suggestion
* WARNING: Split out Event API from Log API. Event API is now published in `opentelemetry-api-events`.
```

Reply:
Good catch